websites/R/Robloxforum/dist/metadata.json

```
@@ -0,0 +1,24 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "sybetjeh.",
+		"id": "963105872408641656"
+	},
+	"service": "Robloxforum",
+	"description": {
+		"en": "A presence for robloxforum! People on discord will be able to see what you do on robloxforum."
```

This could be the description for any presence. It should be a description of the website itself, it's pretty obvious what the presence itself does
```suggestion
		"en": "Welcome to Roblox Forum. Join the discussion here for the best help available. Scripting, Building, Game Creating, or just for fun. We have it all."
```